### ISSUE TITLE
Corrigir o overfitting do modelo de classificação de imagens

### ISSUE THREAD

**Issue by rmmourad** [rmmourad/house_plants#1]
O modelo criado está conseguindo uma acurácia de 0.9186 e loss de 0.2592, mas a val_accuracy e val_loss não estão conseguindo resultados tão positivos.
Isso pode ser um indicativo de overfitting, ou seja, o modelo está performando bem para os dados de treino, mas não tão bem para os de teste. Pode ter diversas causas, e.g. o modelo pode estar 'decorando' as imagens de treinamento.
Esse problema ocorre mesmo utilizando 50% de dropout, então o próximo passo é tentar regularizar os dados ou modificar as imagens para obter um conjunto maior de dados de treinamento. 

![image](https://github.com/user-attachments/assets/342b396d-d922-421a-82e3-6e93f80442bb)
